### PR TITLE
Liberation Station 3: Patriotic Tears

### PR DESCRIPTION
**Pull request by MMMiracles**
image
image

This is the part where I tell you about the liberation station vending machine, the patriotic jumpsuit, and the freedom sheets.

i swear i can do this please no come back im trying i swear

### CODE REVIEW

**Comment by Xhuis** [comment]
It's too late
You've failed
There is no chance for redemption
Just let it happen

:gun: :guardsman:

**Comment by MMMiracles** [comment]
literal tears have been put into this

it can't get any worse by this point can this?

**Comment by as334** [comment]
Don't worry. Freedom never dies.
:us: :us: :us: :firework: :firework: :us: :us: :us: :us: :us:

**Comment by Xhuis** [comment]
:gun: :guardsman: :gb:

**Comment by Cheridan** [comment]
:us: 
:fire: :fearful:

**Comment by ike709** [comment]
:us:

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -14,6 +14,14 @@
 	item_state = "w_suit"
 	can_adjust = 0
 
+/obj/item/clothing/under/patriotsuit
+	name = "Patriotic Suit"
+	desc = "Motorcycle not included."
+	icon_state = "ek"
```
Comment (ike709) [inline code/modules/clothing/under/miscellaneous.dm]: ek

**Comment by MMMiracles** [comment]
this is only the beginning of something greater

just you wait till the 4th of july
:us::us::us::us::us:

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (MMMiracles) [inline code/modules/clothing/under/miscellaneous.dm]: evel knievel shhh

**Comment by PKPenguin321** [comment]
this time it will get through, i believe
:us: :us: :us:

**Comment by ike709** [comment]
yolomerge pls

**Comment by MMMiracles** [comment]
in space nobody can hear the wings of freedom
image

**Comment by pudl** [comment]
dannos literally does a beautiful resprite of the owl and adds a beautiful griffin and now you have to shit it up with a hideous sprite of a hideous thing that's stupid and not funny

**Comment by MMMiracles** [comment]
i mean thats just your opinion but if others aren't cool with it i'll axe it.

**Comment by dannno** [comment]
ahahahahahaha its cool man i dont care

**Comment by MMMiracles** [comment]
alright sweet, good to hear.

**Comment by Chocobro** [comment]
9 more hours

**Comment by MMMiracles** [comment]
clocks ticking

its almost time

**Comment by PKPenguin321** [comment]
:us: :us: :us: :us: :us: :us: :us: :us: :us: :us: :us:

**Comment by MMMiracles** [comment]
:us: god bless america :us:

**Comment by Xhuis** [comment]
:guardsman: :gun:
:fire: :gb: :fire:

**Comment by Iamgoofball** [comment]
>hat uses griffin sprite
>isn't a bald eagle hat

0/10 obviously produced by communist scum

**Pull request merged**